spec/registry.md

```diff
@@ -995,6 +992,7 @@ The function `:date` has these _options_:
   - `long`
   - `medium` (default)
   - `short`
+- _Date/time override options_
```

Comment:
Remove this line.
The overrides right now only support `hour12`, which is not valid on dates.